fetch/mesh_data.go

```diff
@@ -114,18 +113,15 @@ func (f *Fetch) getHashes(
 						zap.Error(p.err),
 					)
 
-					mu.Lock()
-					bfailure.Add(hash, p.err)
-					mu.Unlock()
+					failures.Add(1)
 				}
-				return nil
+				return p.err
```

Comment:
Do we want the `getHashes` request to fail quickly in the case of any hash failing? We could use `errgroup.WithContext` to have the `getHashes` call fail as soon as the first hash fails.

wdyt?

Reply:
No, we don't want the requests for other hashes to fail quickly too. One use case is the ATX sync. We request many missing ATXs. If one is unavailable/invalid, we still want to fetch the rest.